.github/workflows/builds.yml

```diff
@@ -191,3 +191,6 @@ jobs:
         with:
           name: 'development-build'
           path: '${{ github.workspace }}/dss_capi/release/*.zip'
+  image:
+    uses: zepben/.github/.github/workflows/build-sealed-docker-image-public.yml@main
+    secrets: inherit
```

Comment:
TBH, I'd remove this flow as we probably are never going to use it - it is inherited from the original project, which does care to build itself on all platforms, but we don't really.